dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.webAppSecurity-2.0.feature

```diff
@@ -0,0 +1,18 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.webAppSecurity-2.0
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: com.ibm.websphere.security.web; type="ibm-api"
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.servlet-5.0))", \
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.appSecurity-4.0))"
+IBM-Install-Policy: when-satisfied
+-features=com.ibm.websphere.appserver.containerServices-1.0, \
+ io.openliberty.distributedMap-2.0
+-bundles=com.ibm.ws.webcontainer.security.app, \
+ io.openliberty.webcontainer.security.internal; start-phase:=SERVICE_EARLY, \
+ com.ibm.ws.security.appbnd, \
+ io.openliberty.security.authentication.tai.internal
+-jars=com.ibm.websphere.appserver.api.webcontainer.security.app; location:=dev/api/ibm/
+-files=dev/api/ibm/javadoc/com.ibm.websphere.appserver.api.webcontainer.security.app_1.4-javadoc.zip
+kind=noship
+edition=core
```

Comment:
General comment throughout the changes.  noship features should be edition=full

Reply:
Ok. Didn't realize that.